test/e2e/operator_test.go

```diff
@@ -3424,10 +3424,11 @@ func TestUnsupportedConfigOverride(t *testing.T) {
 	t.Parallel()
 
 	var tests = []struct {
-		name, unsupportedConfigOverride, env string
+		name, unsupportedConfigOverride, env, defaultValue, value string
```

Comment:
I think this would make the test easier to follow:  
```suggestion
		name, unsupportedConfigOverride, env, initialValue, updatedValue string
```